### PR TITLE
feat: add google play store reviews Snap-in.

### PR DESCRIPTION
**Pull request by SinghCoder**
ISS-75177
/close #work

### CODE REVIEW

**Review by igniting — APPROVED** [review]

File: 7-googleplaystore-reviews-ingestion/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+version: "1"
```
Comment (altafDevRev) [inline 7-googleplaystore-reviews-ingestion/manifest.yaml:1]: Better to use version 2.

File: 7-googleplaystore-reviews-ingestion/code/src/functions/process_playstore_reviews/index.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+  data: any;
+};
+
+export const defaultResponse: HTTPResponse = {
+  data: {},
+  message: '',
+  success: false,
+};
+
+export class ApiUtils {
```
Comment (altafDevRev) [inline 7-googleplaystore-reviews-ingestion/code/src/functions/process_playstore_reviews/index.ts:24]: Maybe we can add this class in a separate helper file as this file is getting a little big?

File: 7-googleplaystore-reviews-ingestion/manifest.yaml (hunk 1 above)
Comment (SinghCoder) [reply 7-googleplaystore-reviews-ingestion/manifest.yaml:1]: done

File: 7-googleplaystore-reviews-ingestion/code/src/functions/process_playstore_reviews/index.ts (hunk 2 above)
Comment (SinghCoder) [reply 7-googleplaystore-reviews-ingestion/code/src/functions/process_playstore_reviews/index.ts]: done

File: 7-googleplaystore-reviews-ingestion/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Google Playstore reviews to tickets
+
+The Snap-in provides a command `playstore_reviews_process` which allows to fetch last N reviews of your app (id provided in the configuration of Snap-in) and create tickets out of them.
+For each created ticket, it categorizes them into one-of bug, feature_request, feedback, and question, and adds a corresponding tag on the ticket.
+
+If it fails to categorize a ticket, it adds a tag `failed_to_infer_category`.
+
+## Demo
+
+[Demo.webm](demo.webm)
```
[the remark this answers is not in the document]
Comment (SinghCoder) [reply 7-googleplaystore-reviews-ingestion/README.md]: done

File: 7-googleplaystore-reviews-ingestion/demo.webm
Comment (altafDevRev) [inline 7-googleplaystore-reviews-ingestion/demo.webm:1]: Do we want this as we have already embedded it?

File: 7-googleplaystore-reviews-ingestion/code/build.tar.gz
Comment (altafDevRev) [inline 7-googleplaystore-reviews-ingestion/code/build.tar.gz:1]: This can be removed.

**Review by altafDevRev — APPROVED** [review]

File: 7-googleplaystore-reviews-ingestion/demo.webm
Comment (SinghCoder) [reply 7-googleplaystore-reviews-ingestion/demo.webm:1]: removed

File: 7-googleplaystore-reviews-ingestion/code/build.tar.gz
Comment (SinghCoder) [reply 7-googleplaystore-reviews-ingestion/code/build.tar.gz:1]: removed

**Review by starlightknown — APPROVED** [review]

**Pull request merged**